python/tvm/relay/transform.py

```diff
@@ -974,3 +974,40 @@ def visit_var(self, var):
                 else:
                     return var
         return ChangeBatchMutator().visit(func)
+
+def dispatch_global_func(mod, func_name, input_shape, dispatch_func):
```

Comment:
Maybe a more descriptive name since this mutates the module, i.e `register_dispatch` or `add_dispatch_func`.